app/views/product_users/index.html.haml

```diff
@@ -20,22 +20,4 @@
         = text("import_button")
         %span{ style:"display:none" }= f.file_field :file, onchange: "form.submit()"
 
-- if @product_users.nil?
-- elsif @product_users.empty?
-  %p.notice= text("none")
```

Comment:
I think there's an entry for this in `config/locales/views/admin/en.product_users.yml:7` that can be removed now